multitenancy.md

```diff
@@ -1,28 +1,36 @@
-# Multi-Tenancy
+# Multi-tenancy
 
-### Tenants
-The HRI supports the concept of multiple tenants. See the respective glossary page entries for more details on [Tenants](glossary.md#tenant) and [Multi-Tenancy](glossary.md#multitenancy). 
+Health Record Ingestion supports the concept of [**multi-tenancy**](glossary.md#multitenancy), for multiple [tenants](glossary.md#tenant). 
 
-### Data Integrators
-Additionally, data is isolated between Data Integrators. A [**Data Integrator** is the organization](glossary.md#data-integrator) supplying the Health data on behalf of one or more tenants. 
+In Health Record Ingestion, data is isolated between [**Data Integrators**](glossary.md#data-integrator), that is, the organizations that supply health data on behalf of one or more tenants. 
 
-### Data Consumers
-**Data Consumers** are downstream processes (created by some "customer" org) that read data from the HRI. The HRI is designed so that a single data consumer would read data for a single tenant, but it does not prevent a consumer from reading data for multiple tenants. Data Consumers _can see data provided by all Data Integrators_. 
+Customer organizations create **Data Consumers**, which are downstream processes that read data from Health Record Ingestion. The latter is designed so that a single data consumer reads data for a single tenant. However, this does not prevent a consumer from reading data for multiple tenants. Data Consumers **can see data provided by all Data Integrators**. 
+
+Figure 1 shows the color-coded flow of two different tenants through Health Record Ingestion: 
+
+- The **red color** indicates Tenant 1's data. 
+- The **blue color** indicates Tenant 2's data. 
+- Data Integrator B is both red and blue, because it processes data for both tenants.
+
+**Figure 1: Core architecture, multi-tenancy and Health Record Ingestion**
 
----
 ![core-architecture](assets/img/multitenancy.png)
 
-**Note:** currently the Management API uses IBM Functions API keys for authentication, see [HRI Management API Keys](admin.md#hri-management-api-keys) for more details. 
+**Note:** Currently, the Management API uses IBM&reg; Functions API keys for authentication. For more information, see [Setting up Management API keys](admin.md#hri-management-api-keys). 
+
+#### About Event Streams topics
+In Event Streams, there must be at least one topic for every tenant and Data Integrator. To facilitate this, topics are named using the tenant and Data Integrator's name. Example:
+
+**ingest.tenant.data-integrator**
```

Comment:
This looks like a section header, not an example. Either move it back inline with the paragraph or make a code block.